Transbank/Onepay/Onepay.cs

```diff
@@ -25,5 +29,17 @@ public static string SharedSecret
             set => sharedSecret = value ?? throw 
                 new ArgumentNullException("SharedSecret can't be null");
         }
+
+        public static string AppScheme
+        {
+            get => _appScheme;
+            set => _appScheme = value ?? throw new ArgumentNullException(nameof(_appScheme));
+        }
+        
+        public static string CallbackUrl
+        {
+            get => _callbackUrl;
+            set => _callbackUrl = value ?? throw new ArgumentNullException(nameof(_callbackUrl));
```

Comment:
@goncafa Same as above, the `nameof` usage seems wrong.

Reply:
May we could work around with that waym but i think it is not the right code because the ArgumentNullException param is an string paramName.

I will make this change by now.